src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -682,14 +685,17 @@ namespace SimAirServingZones {
 					break;
 				}
 			}
-			if ( ( test == 0 ) && ( AirToZoneNodeInfo(AirSysNum).NumReturnNodes > 0 ) ) { // there, see if it's in the controlled zone info
+			if ( ( test == 0 ) && ( AirToZoneNodeInfo( AirSysNum ).NumReturnNodes > 0 ) ) { // there, see if it's in the controlled zone info
```

Comment:
Hmm... `test` doesn't make a great variable name.  But I know you didn't add it, so no worries.